docs/src/pages/index.tsx

```diff
@@ -273,18 +273,18 @@ export default function Home({ location, path }: Props) {
                 free.
               </BrandedTile>
               <BrandedTile
-                title="Follow us on Twitter"
+                title="Follow us on X"
                 icon={<LoveBirdIcon />}
-                href="https://twitter.com/OrbitKiwi"
-                linkContent="Follow Orbit on Twitter"
-                logo={<TwitterLogo />}
+                href="https://x.com/OrbitKiwi"
+                linkContent="Follow Orbit on X"
+                logo={<XLogo />}
                 color={{
-                  primary: "#0989CF",
-                  secondary: "#179CE3",
+                  primary: "#15171a",
+                  secondary: "#657785",
                 }}
               >
-                Twitter is one of our main platforms for sharing. Everything important that is
-                happening around Orbit is published on Twitter.
+                X is one of our main platforms for sharing. Everything important that is happening
```

Comment:
Honestly, I would just remove this tile entirely. And the Tequila one next to it. We can discuss this within the team.

Reply:
Agreed to remove both IRL